src/node_crypto.cc

```diff
@@ -2563,6 +2564,18 @@ static bool IsSupportedAuthenticatedMode(int mode) {
          IS_OCB_MODE(mode);
 }
 
+static bool IsAEADChacha20Poly1305(const EVP_CIPHER* cipher) {
+  // see openssl/crypto/evp/e_chacha20_poly1305.c#L612
```

Comment:
Please capitalize comments. Also, line numbers can change over time, so this reference might become outdated.

Reply:
OK, I will use nid to check, and this comment is actually no more needed,I will fix it